arches/app/media/js/views/form.js

```diff
@@ -23,10 +23,10 @@ define([
         initialize: function(options) {
             var self = this;
             self.modelReady = ko.observable(false);
-            require(['models/'+(options.modelName?options.modelName:'tile')], function (TileModel) {
-                self.TileModel = TileModel;
-                self.modelReady(true);
-            })
+            // require(['models/'+(options.modelName?options.modelName:'tile')], function (TileModel) {
```

Comment:
requiring `models/...` here is breaking the application, because the absolute path is not being accepted. Changing this to a relative path (`../models/..`) will resolve the error.